kasa/tests/test_usage.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from kasa.modules import Usage
+
+
+def test_usage_convert_stat_data():
+    usage = Usage(None, module="usage")
+
+    test_data = []
+    assert usage._convert_stat_data(test_data, "day") == {}
+
+    test_data = [
+        {"year": 2016, "month": 5, "day": 2, "time": 20},
+        {"year": 2016, "month": 5, "day": 4, "time": 30},
+    ]
+    d = usage._convert_stat_data(test_data, "day")
+    assert len(d) == len(test_data)
+    assert isinstance(d, dict)
+    k, v = d.popitem()
+    assert isinstance(k, int)
```

Comment:
Please test also for the values, just in case the format gets changed at some point it's good to know about it.